includes/html/list/devices.inc.php

```diff
@@ -68,4 +68,6 @@
 
 $more = ($offset + count($devices)) < $total;
 
+array_multisort(array_column($devices, 'text'), SORT_ASC, $devices);
```

Comment:
`text` isn't a valid column name in the `devices` table.

Also, you need to check if `Config::get('force_ip_to_sysname')` is true and use `sysName` for sorting if not, `hostname` should be used.

Reply:
@laf 
"format_hostname()" returns the hostname string, and checks also 'force_ip_to_sysname'

this function will be used in an array_mapping function which returns an array of associative arrays, containing the keys 'id' and 'text'. 'text' contains the return value of "format_hostname()"
Take a look on lin 61-67

so on my views there is nothing to change
(i tested the code and it works well)

Reply:
Good for me, tested all right